doc/ref_arch/openstack/chapters/chapter04.md

```diff
@@ -174,7 +174,16 @@ Octavia depends upon a number of OpenStack services including Nova for spinning
 Octavia supports provider drivers which allows third-party load balancing drivers (such as F5, AVI, etc.) to be utilized instead of the default Amphorae load balancer. When creating a third-party load balancer, the **provider** attribute is used to specify the backend to be used to create the load balancer. The **list providers** lists all enabled provider drivers.  Instead of using the provider parameter, an alternate is to specify the flavor_id in the create call where provider-specific Octavia flavors have been created. 
 
 
-#### 4.2.3.4. Neutron ML2 integration
+#### 4.2.3.4. Neutron Extensions
+OpenStack Neutron is an extensible framework that allows new network capabilities to be incorporated through plugins and API Extensions. API Extensions provides a method for introducing new functionality and vendor specific capabilities. Neutron allows incorporation of plugins which support the new or vendor-specific functionality. Extensions also allow specifying new resources or extensions to existing resources and the actions on these resources.  Plugins implement these resources and actions.
+
+CNTT Reference Architecture support the ML2 plugin (see below) as well as the service plugins including for [FWaaS (Firewall as a Service)[( http://git.openstack.org/cgit/openstack/neutron-fwaas/), [LBaaS (Load Balancer as a Service)](https://governance.openstack.org/tc/reference/projects/octavia.html), and [VPNaaS (VPN as a Service)]( http://git.openstack.org/cgit/openstack/neutron-vpnaas/). The OpenStack wiki provides a list of [Neutron plugins](https://wiki.openstack.org/wiki/Neutron) (jump to “Plugins”).
+
```

Comment:
Direct reference may be given to https://wiki.openstack.org/wiki/Neutron#Plugins instead of ```...list of [Neutron plugins](https://wiki.openstack.org/wiki/Neutron) (jump to “Plugins”).```

Reply:
Done